pallets/file-system/src/utils.rs

```diff
@@ -2398,6 +2398,22 @@ where
 
         Ok((to_succeed, threshold_slope))
     }
+
+    pub fn storage_requests_by_msp(
```

Comment:
Leaving a trace here: We could rename this to `unresponded_storage_requests_by_msp`